compiler/rustc_session/src/options.rs

```diff
@@ -891,6 +891,8 @@ options! {
         "allow the linker to link its default libraries (default: no)"),
     embed_bitcode: bool = (true, parse_bool, [TRACKED],
         "emit bitcode in rlibs (default: yes)"),
+    export_executable_symbols: bool =(false, parse_bool, [TRACKED],
```

Comment:
```suggestion
    export_executable_symbols: bool = (false, parse_bool, [TRACKED],
```
formatting nit. rustfmt doesn't format macro invocations.